_search-plugins/knn/knn-index.md

```diff
@@ -107,25 +162,21 @@ An index created in OpenSearch version 2.11 or earlier will still use the old `e
 {: .note}
 
 ```json
-{
-    "type": "knn_vector",
-    "dimension": 100,
-    "method": {
-        "name":"hnsw",
-        "engine":"lucene",
-        "space_type": "l2",
-        "parameters":{
-            "m":2048,
-            "ef_construction": 245
-        }
+"method": {
+    "name":"hnsw",
+    "engine":"lucene",
+    "space_type": "l2",
+    "parameters":{
+        "m":2048,
+        "ef_construction": 245
     }
 }
 ```
 
-### Supported faiss encoders
+### Supported Faiss encoders
 
-You can use encoders to reduce the memory footprint of a k-NN index at the expense of search accuracy. faiss has
-several encoder types, but the plugin currently only supports *flat* and *pq* encoding.
+You can use encoders to reduce the memory footprint of a k-NN index at the expense of search accuracy. Faiss has
+several encoder types, but the plugin currently only supports `flat`, `pq`, and `sq` encoding.
```

Comment:
<s> Faiss has
several encoder types, but the plugin currently only supports `flat`, `pq`, and `sq` encoding </s>

k-NN plugin currently supports  `flat`, `pq`, and `sq` encoders from Faiss library?.

Reply:
ack